src/Core/Services/IOrganizationSponsorshipService.cs

```diff
@@ -7,7 +7,7 @@ namespace Bit.Core.Services
 {
     public interface IOrganizationSponsorshipService
     {
-        Task<bool> ValidateRedemptionTokenAsync(string encryptedToken);
+        Task<bool> ValidateRedemptionTokenAsync(string encryptedToken, string currentUserEmail);
```

Comment:
We seem to mostly be using `sponsoring` / `sponsored` concept here. `currentUserEmail` feels a bit out of place. Maybe `sponsoredUserEmail`?